docs/api.rst

```diff
@@ -3037,6 +3037,24 @@ of :class:`enum.Enum`.
 
         Sort forum posts by their creation date.
 
+.. class:: ForumLayoutType
+
+    .. versionadded:: 2.4
+
+    The default layout type used to display posts in a :class:`ForumChannel`.
```

Comment:
Usually the first line of docs is the short summary.
```suggestion
    The default layout type used to display posts in a :class:`ForumChannel`.

    .. versionadded:: 2.4
```

Reply:
I could sneak the same thing in for `SortOrderType` if you want.